flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -1989,16 +1989,15 @@ Symbol *OmpAttributeVisitor::ResolveOmpCommonBlockName(
     return nullptr;
   }
   // First check if the Common Block is declared in the current scope
-  if (auto *cur{GetContext().scope.FindCommonBlock(name->source)}) {
-    name->symbol = cur;
-    return cur;
-  }
+  Symbol *sym = GetContext().scope.FindCommonBlock(name->source);
   // Then check parent scope
-  if (auto *prev{GetContext().scope.parent().FindCommonBlock(name->source)}) {
-    name->symbol = prev;
-    return prev;
-  }
-  return nullptr;
+  if (!sym)
+    sym = GetContext().scope.parent().FindCommonBlock(name->source);
+  if (!sym)
+    return nullptr;
```

Comment:
It seems to me that the preferred style in semantics is to always use braces, even with single statements.